docs/Cloud_integrations/webhooks.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Webhooks
+date: 2023-08-02
+og_title: AVSystem IoT Developer Zone
+---
+
+# Webhooks
+
+The webhook event handler delivers HTTP(S) requests to public endpoints. Webhooks contain lifecycle events and/or telemetry data which can be delivered to web applications, IoT platforms, API gateways or hyperscaler lambda functions.
+
+## Prerequisites
+
+* A user with access to the {{ coiote_long_name }} and appropriate permissions
+* A LwM2M device
+
+## Create a webhook
+
+To create a new webhook event handler, select: **Integrations** > **Data Integration Center**.
+
+![Data integration center](images/data-integration-center.png "Data Integration Center")
+
+Select **Add event handler**.
+
+![Add event handler](images/add-event-handler.png "Add event handler")
+
+Select **Webhook**.
+
+![Add webhook](images/webhook.png "Add webhook")
+
+Provide **basic information** such as the name and description, optionally enabling the option to activate the event handler after creation.  
+
+![Basic information](images/basic-information.png "Provide basic information")
+
+**Select the events** which will trigger the HTTP(S) requests.
+
+!!! Info "Event types"
+    **Device events** include:
+
+    - Device created
+    - First registration
+    - {{ coiote_short_name }} writes a value to a device
+    - Device executes a firmware update
```

Comment:
To be 100% precise, this should say `Device successfully executes a firmware update`, because Coiote doesn't (at least, right now) send events when FOTA fails